plugins/optimization-detective/storage/rest-api.php

```diff
@@ -169,28 +172,20 @@ function od_handle_rest_request( WP_REST_Request $request ) {
 	$post_id = $result;
 
 	/**
-	 * Fires whenever a URL Metric was successfully collected.
+	 * Fires whenever a URL Metric was successfully stored.
 	 *
-	 * @since 0.6.0
+	 * @since n.e.x.t
 	 *
-	 * @param array $context {
-	 *     Context about the successful URL Metric collection.
-	 *
-	 *     @var int                                   $post_id
-	 *     @var WP_REST_Request<array<string, mixed>> $request
-	 *     @var OD_Strict_URL_Metric                  $url_metric
-	 *     @var OD_URL_Metrics_Group                  $group
-	 *     @var OD_URL_Metrics_Group_Collection       $group_collection
-	 * }
+	 * @param OD_URL_Metric_Stored_Context $context Context.
```

Comment:
Maybe keep the more precise description from before?
```suggestion
	 * @param OD_URL_Metric_Stored_Context $context Context about the successful URL Metric collection.
```